WordPress/src/main/java/org/wordpress/android/ui/jetpack/scan/ScanViewModel.kt

```diff
@@ -195,11 +197,20 @@ class ScanViewModel @Inject constructor(
         }
     }
 
-    private fun updateFixThreatsStatusProgressBar(isVisible: Boolean) {
+    private fun updateFixThreatsStatusProgressBar(
+        scanStateModel: ScanStateModel,
+        fixingThreatIds: List<Long> = emptyList()
```

Comment:
Question (❓): The `emptyList()` default value is not necessary here, isn't that right? Do you want this function to work without having the caller pass a `fixingThreadIds` in the future? If not, I would consider removing that to avoid setting this expectation.

Reply:
Makes sense, removed in 8951875.